web/src/views/admin/AdminOrgs.vue

```diff
@@ -66,3 +66,9 @@ const { doSubmit: deleteOrg, isLoading: isDeleting } = useAsyncAction(async (_or
   resetPage();
 });
 </script>
+
+<style scoped>
```

Comment:
Same question here, why do we need a scoped class for a single property?

Reply:
I didn't work with overrides or normal injection. Classes collided with ones merged from parent divs and didn't take effect. Scoped classes seem to solve this.
If you know a better way to solve this, please propose it.